apisix/plugins/tcp-logger.lua

```diff
@@ -95,6 +98,23 @@ local function send_tcp_data(conf, log_message)
 end
 
 
+local function remove_stale_objects(premature)
+    if premature then
+        return
+    end
+
+    for key, batch in ipairs(buffers) do
+        if #batch.entry_buffer.entries == 0 and #batch.batch_to_process == 0 then
+            core.log.debug("removing batch processor stale object,"  ..
```

Comment:
bad style: avoid to concat string object

`core.log.debug("remove **", "****", "****")`

Reply:
Thank you, fixed for the udp, kafka as well.